saleor/graphql/shipping/types.py

```diff
@@ -252,9 +258,11 @@ def resolve_tax_class(root: ChannelContext[models.ShippingMethod], info):
 
 
 class ShippingZone(ChannelContextTypeWithMetadata[models.ShippingZone]):
-    id = graphene.GlobalID(required=True)
-    name = graphene.String(required=True)
-    default = graphene.Boolean(required=True)
+    id = graphene.GlobalID(required=True, description="The ID of shipping zone")
+    name = graphene.String(required=True, description="Shipping zone name.")
+    default = graphene.Boolean(
+        required=True, description="The shipping zone by default."
```

Comment:
Maybe something more like
```suggestion
        required=True, description="Indicates if the shipping zone is default one."
```
?